apps/artemis-web/pages/password_change.tsx

```diff
@@ -29,8 +29,8 @@ const PasswordChangePage = (props) => {
   );
 };
 
-export default AuthHOC(PasswordChangePage, ['admin', 'user']);
+export default AuthHOC(PasswordChangePage, ['admin', 'user'], 'ldap-user');
```

Comment:
why is the default 'ldap-user' and not 'user'?

Reply:
Its not a default value :) it just says block  the "ldap-user" !